GameData/RealismOverhaul/RemoteTech_Settings.cfg

```diff
@@ -1,9 +1,14 @@
-RemoteTechSettings:NEEDS[RealismOverhaul]
+//  ==================================================
+//  Custom RemoteTech settings for Realism Overhaul.
+
+//  *   Make "Root" the default communication rule set.
+//  *   Allow combining multiple antennae for range extension.
+//  *   Zero the active engine throttle on connection loss.
```

Comment:
Looks like we have this set to `False`, but the comment implies we have this set to `True`.  I would think the more realistic option is leaving this to `False`, but we should probably update this to say "Don't zero the active engine throttle on connection loss."

> ThrottleZeroOnNoConnection (default = True)
If true, the flight computer cuts the thrust if you have no connection to mission control.